anubis-management-api/anubis/policies/routers.py

```diff
@@ -223,14 +219,13 @@ def read_agent_types(
     },
 }
 
-# TODO if no token, we should return policies for foaf:Agent!
-
 
 @router.get("/me",
```

Comment:
How do we test that if there's no token then we return the all the policies for the "public" agent---`foaf:Agent`?
It looks like the implementation is fine, but do we have a test for it?